lessons/10_conditions/index.html

```diff
@@ -0,0 +1,512 @@
+					<li class="fragment">How can you do it with if?</li>
+				</ul>
+			</section>
+			<section>
+				<h3>Number comparisons</h3>
+				<p class="fragment">So far, we can only check for equality, e.g.</p>
+				<span class="fragment">
+					<pre><code data-trim>
+						let age = prompt("What's your age?");
+						if (age === 19) {
+							// Old enough! But what about 20? 21?
+						}
+					</code></pre>
+				</span>
+			</section>
+
+			<section>
+				<h3>Number comparisons</h3>
+				<ul>
+					<li>How to check for "older than 18?"</li>
+					<li class="fragment"><code>age &gt; 18</code></li>
+					<li class="fragment">The opposite: <code>age &lt; 18</code></li>
+					<li class="fragment">
+						Exercise:
+						<ul class="smaller">
+							<li>Write a function "checkAge(age)"</li>
+							<li>If older than 18, return "adult"</li>
+							<li>If younger than 18, return "child"</li>
+							<li>Try it out with some different ages!</li>
+						</ul>
+					</li>
+				</ul>
+			</section>
+
+			<section>
+				<h3>Number comparisons</h3>
+				<ul>
+					<li>Problem: What about age 18? Should count as adult!</li>
+					<li class="fragment">We need a check "greater than or equal"</li>
+					<li class="fragment">Try: <code>age &gt;= 18</code></li>
+					<li class="fragment">Opposite: <code>age &lt;= 18</code></li>
+					<li class="fragment">Update your function with the correct comparisons!</li>
+				</ul>
+			</section>
+
+			<section class="smaller">
+				<h3>Summary: Comparisons ⚖️</h3>
+				<ul>
+					<li>Comparisons are what we put inside a <code>if(...)</code></li>
+					<li class="fragment">Are used to compare two values</li>
+					<li class="fragment"><code>===</code> <i class="fragment">(equals)</i></li>
+					<li class="fragment"><code>&lt;</code> <i class="fragment">(less than)</i></li>
+					<li class="fragment"><code>&gt;</code> <i class="fragment">(greater than)</i></li>
+					<li class="fragment"><code>&lt;=</code> <i class="fragment">(less than or equal to)</i></li>
+					<li class="fragment"><code>&gt;=</code> <i class="fragment">(greater than or equal to)</i></li>
+					<li class="fragment"><code>!==</code> <i class="fragment">(not equal)</i></li>
+				</ul>
+			</section>
+
+			<section>
+				<h3>Bonus: complex conditions</h3>
+				<ul>
+					<li>What if we have multiple conditions to check against?</li>
+					<li class="fragment">Challenge: Show alert if user is 10+ years old and love cats</li>
+					<li class="fragment">
+					<pre><code data-trim>
+						function showCat(age, loveCats){
+						if (age >= 10 && loveCats === true) {
+							alert('🐱')
+						}
+						}
+						showCat(12, true)
+					</code></pre>
+					</li>
+					<li class="fragment">We say AND with <code>&&</code> </li>
+					<li class="fragment">We say OR with <code>||</code></li>
+					<li class="fragment">
+					<pre><code data-trim>
+						function showCat(age, loveCats){
+						if ((age >= 10 && loveCats === true) || age === 65) {
```

Comment:
What about logical operators? Would you explain those?
Probably we need to provide a link to an external resource to read more about it.

Reply:
Good idea, i will include it in the wiki in teams